packages/graphql/tests/integration/issues/360.int.test.ts

```diff
@@ -0,0 +1,204 @@
+
+        const neoSchema = new Neo4jGraphQL({
+            typeDefs,
+        });
+
+        const query = `
+            query ($rangeStart: DateTime, $rangeEnd: DateTime, $activity: String) {
+                ${pluralType}(where: { OR: [{ start_GTE: $rangeStart }, { start_LTE: $rangeEnd }, { activity: $activity }] }) {
+                    id
+                }
+            }
+        `;
+
+        try {
+            await session.run(
+                `
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime(), end: datetime()})
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime(), end: datetime()})
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime(), end: datetime()})
+                `
+            );
+
+            const gqlResult = await graphql({
+                schema: neoSchema.schema,
+                source: query,
+                contextValue: { driver },
+            });
+
+            expect(gqlResult.errors).toBe(undefined);
+            expect((gqlResult.data as any)[pluralType]).toHaveLength(3);
+        } finally {
+            await session.close();
+        }
+    });
+
+    test("should recreate given test in issue and return correct results", async () => {
+        const session = driver.session();
+
+        const type = `${generate({
+            charset: "alphabetic",
+            readable: true,
+        })}Event`;
+
+        const pluralType = pluralize(camelcase(type));
+
+        const typeDefs = `
+            type ${type} {
+                id: ID!
+                name: String
+                start: DateTime
+                end: DateTime
+                activity: String
+            }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({
+            typeDefs,
+        });
+
+        const rangeStart = new Date().toISOString();
+        const rangeEnd = new Date().toISOString();
+
+        const query = `
+            query ($rangeStart: DateTime, $rangeEnd: DateTime, $activity: String) {
+                ${pluralType}(where: { OR: [{ start_GTE: $rangeStart }, { start_LTE: $rangeEnd }, { activity: $activity }] }) {
+                    id
+                }
+            }
+        `;
+
+        try {
+            await session.run(
+                `
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime($rangeStart), end: datetime($rangeEnd)})
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime($rangeStart), end: datetime($rangeEnd)})
+                    CREATE (:${type} {id: randomUUID(), name: randomUUID(), start: datetime(), end: datetime()})
+                `,
+                { rangeStart, rangeEnd }
+            );
+
+            const gqlResult = await graphql({
+                schema: neoSchema.schema,
+                source: query,
+                contextValue: { driver },
+            });
```

Comment:
If looking to reproduce the scenario in #360, this is missing `variableValues`, and the expected results will need changing.

```suggestion
            const gqlResult = await graphql({
                schema: neoSchema.schema,
                source: query,
                contextValue: { driver },
                variableValues: { rangeStart, rangeEnd },
            });
```